generators/server/files.js

```diff
@@ -893,6 +893,30 @@ const serverFiles = {
             templates: ['config/bootstrap.yml', 'config/bootstrap-prod.yml'],
         },
     ],
+    serverMicroserviceAndOauth: [
+        {
+            condition: generator =>
+                !generator.reactive && generator.authenticationType === 'oauth2' && generator.applicationType === 'microservice',
```

Comment:
This conditional (and others) should include `applicationType === 'monolith'` and `gateway` too. We configure these as resource servers by default for Ionic and React Native apps.

Reply:
@Falydoor I believe this comment still needs to be addressed.